src/persist-localstorage-experimental/index.ts

```diff
@@ -29,15 +36,32 @@ export function persistWithLocalStorage(
     throttleTime = 1000,
     maxAge = 1000 * 60 * 60 * 24,
     buster = '',
+    blacklistQueries = [],
+    blacklistMutations = [],
   }: Options = {}
 ) {
   if (typeof window !== 'undefined') {
     // Subscribe to changes
     const saveCache = throttle(() => {
       const storageCache: LocalStorageCache = {
+        const cacheState = dehydrate(queryClient, {
+            shouldDehydrateQuery: (query) => {
+                const isExist =
+                    blacklistQueries.filter((bl) => deepEqual(bl, query.queryKey))
```

Comment:
I’m pretty sure there are Util functions already to compare query keys without the need of an extra dependency :)

Reply:
Indeed. We should use those utils